crypto/rsa/blinding.c

```diff
@@ -317,16 +316,12 @@ BN_BLINDING *rsa_setup_blinding(const RSA *rsa, BN_CTX *ctx) {
   assert(ctx != NULL);
   assert(rsa->mont_n != NULL);
 
-  BIGNUM local_n;
-  BIGNUM *n;
-
   if (rsa->e == NULL) {
     OPENSSL_PUT_ERROR(RSA, RSA_R_NO_PUBLIC_EXPONENT);
     return NULL;
   }
 
-  n = &local_n;
-  BN_with_flags(n, rsa->n, BN_FLG_CONSTTIME);
+  assert(BN_get_flags(rsa->n, BN_FLG_CONSTTIME) != 0);
```

Comment:
Let's use a consistent style: either we omit the `!= 0` always or we include it always. My thought is that we should just omit it always.